posthog/src/test/java/com/posthog/PostHogTest.kt

```diff
@@ -33,7 +33,7 @@ internal class PostHogTest {
     private val serializer = PostHogSerializer(PostHogConfig(API_KEY))
     private lateinit var config: PostHogConfig
 
-    private val file = File("src/test/resources/json/basic-decide-no-errors.json")
+    private val file = File("src/test/resources/json/basic-decide-v3-no-errors.json")
```

Comment:
maybe we just move these files into a v3 folder? its a bit more organized than a v3/v4 suffix in all files, not a blocker though, just an idea.

Reply:
Good call!